### DISCUSSION TITLE
Fine tuning for beginner

### DISCUSSION THREAD

**Discussion by gtamas** [fishaudio/fish-speech#1278] [Q&A]
My goal is to create a voice for my character.
Does it make sense to fine tune a s2-pro model for this or should I just use some audio reference files and generate voice like that?

I don't have a huge dataset, just 5 mins of clear speech. It's just one speaker and speaking with different emotions.

Does it make sense to fine for such tiny data set? Should I use more training audio?

What is the ideal number of steps, learning rate for this?

**Comment by gtamas** [comment]
By the way, I can start up  fish-speech with a fine-tuned checkpoint, but do  I still need an audio reference file? I guess, the point would be that now I don't need one, since the model is supposed to speak using my characters voice? Is that correct?

**Reply by fluxlab00-stack** [reply]
You cannot use a 5 mins training data for the model to adopt, it's less likely to work. But if you can find so many similar styles of the same voices like a minimum of 2 hours then you are close. Notwithstanding you can give it a try. If it really matters, I can help you out +234 903 044 3821
